.github/BOTMETA.yml

```diff
@@ -369,6 +369,10 @@ files:
   $modules/notification/rocketchat.py:
     ignored: ramondelafuente
     maintainers: Deepakkothandan
+  $modules/notification/snow_record.py:
+    migrated_to: servicenow.servicenow
```

Comment:
Thanks, though I don't see servicenow.servicenow on Galaxy. Have they been published yet?

Reply:
https://galaxy.ansible.com/servicenow/servicenow

Reply:
Thanks.
Can you please add module_utils and the inventory file to botmeta.yml as well.

Reply:
Added. Inventory was never part of core (#63912) so it will just be new the collection.